### PR TITLE
Merge tag can now be used for Activity layouts

### PR DESCRIPTION
**Pull request by Rolf-Smit**
This approach is simpeler and probably more appropriate than the approach suggested in pull-request: [#35](https://github.com/palaima/DebugDrawer/pull/35)

- The `DebugDrawer.Builder` was only placing the first child from the root content view to the DebugDrawer content view, other child's where forgotten, a typical Christmas story 😉🎄
- Made the "alreadyInflated" check a bit more robust
- Updated Gradle build tools to version 2.3.1 (stable)

**TLDR:** This fix makes it possible to use the merge tag for an activity layout.

### CODE REVIEW

**Comment by Rolf-Smit** [comment]
For people having the merge tag issue a temporary work around is quite simple:

**Before building the DebugDrawer:**

            /*
            The current version (0.7) of the DebugDrawer library prevents us from using the merge
            tag as root tag for activity layouts. The DebugDrawer only stores the first view added
            to the root content view (android.R.id.content) the rest of the views are ignored which
            is incorrect and probably a bug.

            This temporary fix stores those forgotten views and adds them after the DebugDrawer has
            done it's magic.
            
            As soon as this pull-request is merged and released this fix can be removed.
            https://github.com/palaima/DebugDrawer/pull/79
             */
            final ViewGroup rootView = (ViewGroup) activity.findViewById(android.R.id.content);
            final ArrayList<View> forgottenContentViews = new ArrayList<>(rootView.getChildCount() - 1);
            for(int i = 1; i < rootView.getChildCount(); i++){
                forgottenContentViews.add(rootView.getChildAt(i));
            }

**After building the DebugDrawer:**

            final ViewGroup debugDrawerRootView = (ViewGroup) rootView.findViewById(R.id.dd_content_layout);

            //If not in debug mode but in release mode the debugDrawerRootView is null (because the 
            DebugDrawer does not actually do anything).
            if(debugDrawerRootView != null){

                //create the layoutParams to use for the contentView
                FrameLayout.LayoutParams layoutParamsContentView = new FrameLayout.LayoutParams(
                        ViewGroup.LayoutParams.MATCH_PARENT,
                        ViewGroup.LayoutParams.MATCH_PARENT
                );

                //add the original content Views to the drawer content frameLayout
                for(View contentView: forgottenContentViews){
                    debugDrawerRootView.addView(contentView, layoutParamsContentView);
                }
            }

**Comment by palaima** [comment]
Thanks for PR. I have included these changes to new upcoming release.

**Comment by Rolf-Smit** [comment]
Question: normally open-source project contributions are merged into the main project when accepted. But you seem to close/reject pending pull-requests and instead you are copying the changes to the main project yourself. This way contributions from other people never end-up in the actual project, I personally find this quite rude (people are contributing but cannot take credit for it).

Why are you doing this/using this approach?
